src/main/java/build/buildfarm/server/AdminService.java

```diff
@@ -180,6 +182,14 @@ public void deregisterWorker(
     }
   }
 
+  @Override
+  public void disableScaleInProtection(
+      DisableScaleInProtectionRequest request,
+      StreamObserver<DisableScaleInProtectionRequestResults> responseObserver) {
+    // adminController.disableHostProtection(instance_id, asg_name);
```

Comment:
"host protection" vs "scale in" protection? Not sure what this is

Reply:
aws call it scale in protection, because that's what it is. Host protection sounds like a more general protection, but I don't exactly what it is.

Reply:
Yeah, i will make the names more consistent here